contrib/dockerd-rootless-setuptool.sh

```diff
@@ -264,6 +264,19 @@ init() {
 		echo
 		exit 1
 	fi
+	if [ -n "$SYSTEMD" ] && ! systemctl --user -q is-active dbus; then
+		# dbus should be automatically enabled on installing dbus-user-session package
+		# (depended by docker-ce-rootless-extras), but dbus does not start automatically
+		# until the next login.
```

Comment:
Oh! Because of this? That's odd; how's that normally supposed to be done (require logout and login again?)

Reply:
Now `systemctl --user start docker` starts dbus